### ISSUE TITLE
why the loss = 0 ,accuracy =1 Minibatch error = 0.0%? 

### ISSUE THREAD

**Issue by sanersbug** [jakeret/tf_unet#201]
When i start the train ,the terminal always output like below:

2018-07-28 20:06:05,497 Epoch 6, Average loss: 0.0000, learning rate: 0.0010
2018-07-28 20:06:07,193 Verification error= 0.0%, loss= 0.0000
2018-07-28 20:06:11,507 Iter 224, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:15,188 Iter 226, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:18,837 Iter 228, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:22,524 Iter 230, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:26,163 Iter 232, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:29,827 Iter 234, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:33,610 Iter 236, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:37,350 Iter 238, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:41,251 Iter 240, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:45,052 Iter 242, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:48,792 Iter 244, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:52,502 Iter 246, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:06:56,236 Iter 248, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:07:00,205 Iter 250, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:07:03,906 Iter 252, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%
2018-07-28 20:07:07,554 Iter 254, Minibatch Loss= 0.0000, Training Accuracy= 1.0000, Minibatch error= 0.0%

Any one knows the reason ? Thanks a lot!

**Comment by sanersbug** [comment]
@jakeret   ,  Finaly , I find the reason .   I have one class , but i changed the 'n_classs=1' , after i modify it , the loss , accuracy... is normal .
